src/um/winnt.rs

```diff
@@ -8378,3 +8378,26 @@ pub const ACTIVATION_CONTEXT_SECTION_COMPATIBILITY_INFO: ULONG = 11;
 STRUCT!{struct ACTIVATION_CONTEXT {
     dummy: *mut c_void,
 }}
+
+pub const LOGON32_LOGON_INTERACTIVE: ULONG = 2;
+pub const LOGON32_LOGON_NETWORK: ULONG = 3;
+pub const LOGON32_LOGON_BATCH: ULONG = 4;
+pub const LOGON32_LOGON_SERVICE: ULONG = 5;
+pub const LOGON32_LOGON_UNLOCK: ULONG = 7;
+//#if(_WIN32_WINNT >= 0x0500)
+pub const LOGON32_LOGON_NETWORK_CLEARTEXT: ULONG = 8;
+pub const LOGON32_LOGON_NEW_CREDENTIALS: ULONG = 9;
+//#endif // (_WIN32_WINNT >= 0x0500)
+
+pub const LOGON32_PROVIDER_DEFAULT: ULONG = 0;
+pub const LOGON32_PROVIDER_WINNT35: ULONG = 1;
+//#if(_WIN32_WINNT >= 0x0400)
+pub const LOGON32_PROVIDER_WINNT40: ULONG = 2;
+//#endif /* _WIN32_WINNT >= 0x0400 */
+//#if(_WIN32_WINNT >= 0x0500)
+pub const LOGON32_PROVIDER_WINNT50: ULONG = 3;
+//#endif // (_WIN32_WINNT >= 0x0500)
+//#if(_WIN32_WINNT >= 0x0600)
+pub const LOGON32_PROVIDER_VIRTUAL: ULONG = 4;
+//#endif // (_WIN32_WINNT >= 0x0600)
```

Comment:
All of these constants are found in `winbase`, not `winnt` in the 10.0.16299.0 SDK

Also extraneous newlines